File: programmers/lev1/12919.py
def solution(seoul):
    answer = ''
    num=0
    for i in range(len(seoul)):
        if seoul[i] =="Kim":
            num=i
            break
    answer="김서방은 "+str(i)+"에 있다"
    return answer

seoul=["Jane","Kim"]
solution(seoul)

File: java/src/programmers/year2023/lv1/_12919.java
package programmers.year2023.lv1;

public class _12919 {
    public static void main(String[] args) {

    }

    public static String solution(String[] seoul) {
        String answer = "";
        for (int i = 0; i < seoul.length; i++) {
            if (seoul[i].equals("Kim")) {
                return "김서방은 " + i + "에 있다";
            }
        }
        return answer;
    }
}
